File: Java/DailyFlash/August/21August/Program3.java


import java.util.Scanner;

class Employee {

	public static void main(String[] args) {

		Scanner sc = new Scanner(System.in);

		int age;
		char sex, maritalStatus;

		System.out.print("Enter the age, sex and marital status(Y/N) ");
		age=sc.nextInt();
		sex=sc.next().charAt(0);
		maritalStatus=sc.next().charAt(0);

		if(sex=='F')
			System.out.println("U will work only in urban ares.");

		else if(sex=='M' && (age>=20 && age<=40))
			System.out.println("U can work anywhere.");

		else if(sex=='M' && (age>40 && age<=60))
			System.out.println("U will work in urban areas only");

		else
	       		System.out.println("Error!");
	}
}	

File: PPA/DailyFlash/August/21August/Program3.c


#include<stdio.h>

void main() {

	int age;
	char sex, maritalStatus;

	printf("Enter your age, sex(M/F) and marital tatus(Y/N): ");
	scanf("%d %c %c",&age,&sex,&maritalStatus);

	if(sex=='F')
		printf("You will work in Urban areas only.\n");
	else if(sex=='M' && (age>=20 && age<=40))
		printf("You can work anywhere.\n");
	else if(sex=='M' && (age>40 && age<=60))
		printf("You will work in Urban areas only.\n");
	else
		printf("Error!\n");
}
